config/routes.rb

```diff
@@ -37,7 +37,8 @@
   namespace :setup do
     get "/", action: :welcome
     match "/", action: :configure, via: [:put, :patch]
-    get :"worker-bootstrap"
+    get  "worker-bootstrap"
+    post "worker-bootstrap", action: :build_cloud_cluster
```

Comment:
maybe have a more descriptive route e.g. `trigger-cloud` or sth similar? i guess you could just tell the form to POST to another route via `url:` argument

Reply:
I changed it in d15b5f90.